_storage/write-to-read.md

```diff
@@ -63,3 +63,16 @@ demonstrate that they control _that_ address. This is achieved via the
 authentication token which is a message _signed_ by the private key associated
 with that address. The message itself is a challenge text, returned via the
 `/hub_info/` endpoint.
+
+Reads can be done by everybody. The URLs to a user's app data are in a canonical location in their profile. For example, here's how you would get data from the
```

Comment:
What do you mean by everybody?  Do you mean non-developers such as end-users? If end user, why would they want to do this? Most of our app users are not bash users... I'm trying to see the use case you are documenting.

Reply:
Everybody in contrast to the owner of the private keys.

The use case is
* for developers to verify what data was written and to understand the difference between read and write.

Reply:
* for end users to see the data they own.

Not sure whether that makes sense for the normal end-user. Maybe we can just show two screenshots of banter message as shown by the app and as stored on gaia.

Reply:
@friedger thanks for clarifying the use case. I think it makes sense for developers to know curl and use it.  Let me review it again with that in mind.